recipes/openfx/all/cmake/OpenFX.cmake

```diff
@@ -0,0 +1,22 @@
+# Add a new OFX plugin target
+# Arguments: TARGET
+function(add_ofx_plugin TARGET)
+	add_library(${TARGET} SHARED)
+	set_target_properties(${TARGET} PROPERTIES SUFFIX ".ofx" PREFIX "")
+
+	if(NOT DEFINED OFX_SUPPORT_SYMBOLS_DIR)
+		if (NOT DEFINED CONAN_LIB_DIRS_OPENFX)
+			message(FATAL_ERROR "Define OFX_SUPPORT_SYMBOLS_DIR to use add_ofx_plugin().")
+		endif()
+		set(OFX_SUPPORT_SYMBOLS_DIR ${CONAN_LIB_DIRS_OPENFX}/symbols)
+	endif()
+
+	# Add extra flags to the link step of the plugin
+	if(APPLE)
+		set_target_properties(${TARGET} PROPERTIES LINK_FLAGS "-bundle -fvisibility=hidden -exported_symbols_list,${OFX_SUPPORT_SYMBOLS_DIR}/osx.symbols")
```

Comment:
Is not possible to export those compiler flags in cpp_info, instead of exporting this cmake file?

Reply:
This CMake file can be used to create OFX plugin target outside of this recipe.